### PR TITLE
EDGAR 24.2 preview

### PR DESCRIPTION
**Pull request by hermfischer-wf**
#### Reason for change
Preview of EDGAR 24.2, planned for production weekend of June 28th.

#### Description of change
New form type SBSEF

Companion [EdgarRender Pull Request](https://github.com/Arelle/EdgarRenderer/pull/106)

#### Steps to Test
Conf suite

**review**:
@Arelle/arelle

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: arelle/plugin/validate/EFM/Filing.py [hunk 1]
```diff
@@ -4953,11 +5019,7 @@ def getMonetaryDescendants(fromConcept, elr, result, visited=None):
             modelXbrl.warning(f"{dqcRuleName}.{id}",
                               f"Validation was unable to complete rule {dqcRuleName} due to an internal error.  This is not considered an error in the filing.",
                               modelObject=modelXbrl)
-            modelXbrl.debug(
-                "arelle:dqcrtException",
-                _("An unexpected exception occurred in DQCRT"),
-                traceback=traceback.format_exception(*sys.exc_info())
-            )
+            modelXbrl.debug("arelle:dqcrtException", _("DQCRT Exception traceback: {}").format(traceback.format_exception(*sys.exc_info())))
```
Comment (brettkail-wk) [inline arelle/plugin/validate/EFM/Filing.py:5022]: This appears to revert the changes from #1204, which seems wrong.

**Review by austinmatherne-wk — APPROVED** [review]
